engine/src/main/java/org/hibernate/validator/cfg/context/AnnotationProcessingOptions.java

```diff
@@ -13,10 +13,13 @@
  * @author Hardy Ferentschik
  */
 public interface AnnotationProcessingOptions<C extends AnnotationProcessingOptions<C>> {
+
 	/**
 	 * Specifies that annotations specified on the configured type or property should be ignored.
 	 *
 	 * @return Returns itself for method chaining.
+	 * @deprecated Use {@link AnnotationIgnoreOptions#ignoreAnnotations(boolean)} instead.
 	 */
+	@Deprecated
```

Comment:
Yes, check out e.g. the new test `testIgnoreAnnotationsOnMethodAndParameter()`. In XML, "ignore-annotations" can be given at several levels in the tree, e.g. being set to true on the entity level, to false for a specific method and to true for a specific parameter of that method. The API seeks to resemble that, so `ignoreAnnotations()` doesn't quite cut it, as you need to be able to set it to false, too.